README.md

```diff
@@ -9,26 +9,42 @@ A practical functional library for Javascript programmers.
 [![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/ramda/ramda?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 
-Goals
------
+Why Ramda?
+----------
 
 <img src="http://ramda.jcphillipps.com/logo/ramdaFilled_200x235.png" 
-     width="200" height="235" align="left" hspace="12" />
-Using this library should feel as much like using Javascript as 
-possible.  Of course it's functional Javascript, but we're not 
-introducing lambda expressions in strings, we're not borrowing consed 
+     width="200" height="235" align="right" hspace="12" />
+
+Ramda makes it simple for you to build complex logic through
+functional composition. This means sticking simple functions together.
+It is designed to help you get you job more elegantly with simpler code.
+Ramda does not mutate your state. Any data you throw at Ramda is safe from
+hidden state changes, and this is a very powerful weapon to battle complexity.
+
+[Compare Ramda to other FP libraries](http://www.google.com)
+
+While LoDash or Underscore are great libraries, they sometimes introduce
+complexities in your code which Ramda avoids with a simpler design.
```

Comment:
also, i don't think this is fair. `_` is clearly very useful to a lot of people. and complexity can be introduced using any tool.